cardano-testnet/src/Testnet/Start/Cardano.hs

```diff
@@ -282,8 +303,9 @@ cardanoTestnet
         , "--database-path", nodeDataDir </> "db"
         ]
         <> spoNodeCliArgs
-        <> testnetNodeExtraCliArgs nodeOptions
+        <> maybe [] extraCliArgs nodeOptions
     pure $ eRuntime <&> \rt -> rt{poolKeys=mKeys}
+    -- TODO log the node's pid in a file. This is useful for killing the node later.
```

Comment:
If this is deserving of an issue can you create it?

Reply:
Follow-up issue created: https://github.com/IntersectMBO/cardano-node/issues/6149